### ISSUE TITLE
Remove visibility property in Button component

### ISSUE THREAD

**Issue by drillprop** [Joystream/atlas#3337]
Currently, for some unknown reason, we're setting `visibility:hidden` on the element called BorderWrapper and we change that to `visible` depending on context. This property is causing accessibility issues and makes our buttons not accessible, even if they have text inside.

- [ ] Remove `visibility: hidden` property
- [ ] Make sure, that all the variants look the same. Above all check the `text` variants. If we remove the visibility property they look like this:
![image](https://user-images.githubusercontent.com/51168865/195364649-78b58e6f-b68a-42b6-b5b4-6e939c525cdd.png)
And they should:
![image](https://user-images.githubusercontent.com/51168865/195364537-db438e2b-4b96-452a-aaf8-8bb6127fa440.png)
 Line under text should only be shown when hovering or focusing.
